device/mellanox/x86_64-mlnx_msn2700-r0/platform.json

```diff
@@ -0,0 +1,326 @@
+{
+    "chassis": {
+        "name": "MSN2700",
+        "components": [],
+        "modules": [],
+        "fans": [
```

Comment:
just have number of fans.

Reply:
As discussed offline, we will modify the file as such:

1. Remove "modules", as we cannot assume which modules will be present
2. For now, all devices will only specify a "name" attribute, except PSUs and fan drawers, which will also specify a number of fans.

Reply:
fixed